pynetbox-stubs/core/app.pyi

```diff
@@ -1,9 +1,16 @@
+from pynetbox._gen.bgp import BgpApp
+
 from typing import TYPE_CHECKING
 
 if TYPE_CHECKING:
-    from pynetbox.api import Api
+    from pynetbox.core.api import Api
 
 class App:
     def __init__(self, api: "Api", name: str):
         self.api = api
         self.name = name
+
+class PluginsApp:
+    def __init__(self, api: "Api"):
+        self.api = api
+        self.bgp = BgpApp(self.api, "bgp")
```

Comment:
Very good question. This means, everyone who want to use this repository needs their own generated stub files. 

You think we can avoid that by somehow proving a generic "catches all" variant here? See https://github.com/netbox-community/pynetbox/issues/333 for the upstream issue.

Reply:
Added the override of `__getattr__()`.